### PR TITLE
add 2 translations to messages_de.properties for federatedIdentity...

### LINKED ISSUE #28035: update for messages_de.properties required
### Description

There are 2 translation parameters missing in the message file for the German translation

translation for  federatedIdentityUnavailableMessage and federatedIdentityUnmatchedEssentialClaimMessage not exist

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by TuxMK**
add 2 translations to messages_de.properties for federatedIdentity...

add translation for federatedIdentityUnavailableMessage and federatedIdentityUnmatchedEssentialClaimMessag in messages_de.properties

Closes #28035

### CODE REVIEW

**Comment by ahus1** [comment]
@TuxMK - thank you for this PR. Please sign-off your commit as outlined in `CONTRIBUTING.md` before this PR can be merged. Thanks!

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 1]
```diff
@@ -208,6 +208,8 @@ usernameExistsMessage=Benutzername existiert bereits.
 emailExistsMessage=E-Mail existiert bereits.
 
 federatedIdentityExistsMessage=Ein Benutzer mit {0} {1} existiert bereits. Bitte melden Sie sich an der Benutzerkontoverwaltung an um den Benutzer zu verknüpfen.
+federatedIdentityUnavailableMessage=Der mit dem Identitätsanbieter {1} authentifizierte Benutzer {0} ist nicht vorhanden. Bitte wenden Sie sich an Ihren Administrator.
+federatedIdentityUnmatchedEssentialClaimMessage=Das von dem Identitätsanbieter ausgestellte ID-Token entspricht nicht der konfigurierten mind. Anforderrung. Bitte kontaktieren Sie Ihren Administrator.
```
Comment (ahus1) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:212]:
```suggestion
federatedIdentityUnmatchedEssentialClaimMessage=Dem vom Identitätsanbieter ausgestellten ID-Token fehlen die passenden Claims. Bitte kontaktieren Sie Ihren Administrator.
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
I found a typo in `Anforderrung`. While reading it, I came up with this different translation. I think `Claim` should not be translated, let me know if you think otherwise.

**Review by ahus1 — APPROVED** [review]
Thank you for updating the translation!

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties (hunk 1 above)
Comment (TuxMK) [reply themes/src/main/resources-community/theme/base/login/messages/messages_de.properties]: Thanks for your feedback - i have adopted your suggestions

**Pull request merged**